components/dashboard/src/data/prebuilds/prebuild-logs-emitter.ts

```diff
@@ -45,8 +45,6 @@ export function usePrebuildLogsEmitter(prebuildId: string, taskId?: string) {
                 throw new ApplicationError(ErrorCodes.NOT_FOUND, `Prebuild ${prebuildId} not found`);
             }
 
-            setIsLoading(true);
```

Comment:
It seems like `setIsLoading` is not called anymore - can we drop it alltogether?

Reply:
Totally, let's get rid of these.